lib/preload.js

```diff
@@ -561,7 +563,7 @@ class Preloader extends EventEmitter {
 		headers = { ...headers };
 		headers['Authorization'] = `Bearer ${registryToken}`;
 		return await request({
-			url: `https://${this.balenaSettings.registry2Url}${endpoint}`,
+			url: `https://${registryUrl}${endpoint}`,
```

Comment:
In addition to this one, there is another occurrence of `registry2Url` that I think needs amending:
https://github.com/balena-io-modules/balena-preload/blob/v10.4.6/lib/preload.js#L737